### PR TITLE
Refactoring services

### PR DESCRIPTION
**Pull request by Taluu**
Let's update to Symfony 3.3 (FQCN ids, autowiring, autoconfiguring, and some cleanups too)

- [x] FQCN ids
- [x] Mark some services as private (even though it's a break, these services have no sense whatsoever to be used from the container)
- [x] Rename oldsound to phpamqplib
- [x] Add some autowiring, autoconfiguring

### CODE REVIEW

File: src/Resources/config/profiler.xml [hunk 1]
```diff
@@ -5,12 +5,15 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="wisembly.amqp.collector" class="Wisembly\AmqpBundle\DataCollector\AMQPDataCollector">
-            <tag name="data_collector" template="WisemblyAmqpBundle:Collector:MessageCollector" id="amqp_collector" />
-        </service>
+        <defaults public="false" />
```
Comment (Taluu) [inline src/Resources/config/profiler.xml]: note ; not sure if this is correct, as a while ago all service subscribers couldn't be private. Don't know if it is still the case...

Comment (Taluu) [reply src/Resources/config/profiler.xml]: it's fiiiiiine : https://github.com/symfony/symfony/pull/20953/files#diff-8604f72b4ae78c6c364c1af8c1dee17cL62

File: src/Resources/config/amqp.xml [hunk 2]
```diff
@@ -5,11 +5,13 @@
...
     <services>
-        <service id="wisembly.amqp.publisher" class="Wisembly\AmqpBundle\Publisher">
-            <argument type="service" id="event_dispatcher" />
-            <argument type="service" id="wisembly.amqp.broker" /> <!-- declared in a compiler pass //-->
-        </service>
+        <defaults public="false" autowire="true" autoconfigure="true" />
 
-        <service id="wisembly.amqp.gates" class="Wisembly\AmqpBundle\GatesBag" />
+        <service id="Wisembly\AmqpBundle\Publisher" public="true" />
+        <service id="Wisembly\AmqpBundle\GatesBag" />
```
Comment (lunika) [inline src/Resources/config/amqp.xml]: why this service is private and its alias public ?

Comment (Taluu) [reply src/Resources/config/amqp.xml]: backward compatibility

**Comment by Taluu** [comment]
@lunika says it works fine (while I had segfaults..) so it should be fine. o/

**Pull request merged**